### ISSUE TITLE
Add better usage instructions

### ISSUE THREAD

**Issue by JonathanReeve** [Antergos/antergos-iso#72]
I can't figure out how to use these scripts for the life of me. I've run `make install` and copied over the files (which I think are the same ones in config anyway?), but then I don't know what to do next. I tried running `config/antergos/build.sh single` but it doesn't seem to work. Could you add usage instructions to the README so that the method of making an ISO using these scripts is more clear?

**Comment by karasu** [comment]
It's been a while since I last build an ISO (@lots.0.logs coded a fantastic build server that does this automagically). Have you tried running it inside the antergos folder? Also, instead of `single` I remember using `dual`, but I'm not sure if it will make a difference.

**Comment by Wyn10** [comment]
@karasu It was asked about here: https://forum.antergos.com/topic/5103/

I threw a notification for @lots0logs about it but I don't think he saw it.

**Comment by lots0logs** [comment]
Actually I did see it, but unfortunately have not had any time to respond :grimacing:

**Comment by Wyn10** [comment]
@lots0logs 😢

**Comment by Zeioth** [comment]
If i run
    git tag v3
Then I can run 
    sudo make dist

And it generates a "antergos-iso-3.tar.gz". But it's only 15,4Mb. I tried to convert it to ISO but it's not bootable. 

Please, add at least some basic knowledge about how to edit the ISO and make it bootable. I'd really like to create a nice XFCE theme like Xubuntu or Manjaro.

**Comment by lots0logs** [comment]
Have you checked [this page](https://wiki.archlinux.org/index.php/archiso) on the Arch wiki? The thing about our ISO script currently it is configured in a way that it expects it is being run from our server. Unless you are comfortable with shell scripting you will have a hard time getting it to work locally. At some point we will rewrite the scripts so that they can be used anywhere. I just dont know when that will be exactly. Sorry! :grimacing:

**Comment by Zeioth** [comment]
You are right. I've been tweaking a  couple of hours. This is as far as I've come. I'll continue tomorrow.

![](http://i.imgur.com/d0XBFjv.png)

**Comment by karasu** [comment]
- The correct command should be `sudo ./build.sh build dual`
- Remember to download openfonts (or comment that line in build.sh).
- You will need to also pull antergos-gfxboot (or use isolinux).

That's all I can recall atm.

**Comment by philwyettreb** [comment]
I have what appears to be a successfully built local ISO that I am installing in a vm from.

Further investigation needed as:

- Build instructions are a little wrong.
- Generated ISO is 600M smaller than current release one.
- What is this transifex token crap.

**Comment by philwyettreb** [comment]
Quick rundown of what I have so far.

Add / Install dep:

gfxboot

Diff of build.sh:

```
[philwyett@rebellion antergos]$ diff -u build.sh.orig build.sh
--- build.sh.orig	2017-02-21 06:28:03.439288654 +0000
+++ build.sh	2017-02-21 05:09:30.024548630 +0000
@@ -10,7 +10,7 @@
 install_dir="arch"
 arch=$(uname -m)
 work_dir=work
-out_dir=/out
+out_dir=out
 verbose="-v"
 cmd_args=""
 keep_pacman_packages="y"
@@ -216,8 +216,8 @@
         	#mkdir -p ${work_dir}/root-image/var/run/dbus
         	#mount -o bind /var/run/dbus ${work_dir}/root-image/var/run/dbus
         	# Download opendesktop-fonts
-        	#wget --content-disposition -P ${work_dir}/root-image/arch/pkg 'https://www.archlinux.org/packages/community/any/opendesktop-fonts/download/'
-        	cp /start/opendesktop**.xz ${work_dir}/root-image/arch/pkg
+        	wget --content-disposition -P ${work_dir}/root-image/arch/pkg 'https://www.archlinux.org/packages/community/any/opendesktop-fonts/download/'
+        	#cp /start/opendesktop**.xz ${work_dir}/root-image/arch/pkg
         	touch /var/tmp/one
         }
 
[philwyett@rebellion antergos]$
```

Create 'out' folder in the 'antergos' folder. sudo not required and no need to be on /.
Create the 'start' symbolic link in 'antergos folder. sudo not required and no need to be on /.
'cd' into ' antergos-gfxboot'. Build gfxboot isolinux data by executing 'make'. After, copy the generated 'isolinux' folder into the 'antegos' folder.
Run 'sudo ./build.sh build dual'.

**Comment by karasu** [comment]
@philwyettreb I like changing /out and /start to out and start, no need to be in / (I suppose it's related to the build server).

Have you seen the testing branch? I've changed the build.sh script to work ootb for all versions except of the kde one.

That iso is too small. I think you're removing the xz packages from it. It's ok, but this means that the installation will take more time.

**Comment by philwyettreb** [comment]
@karasu Just built the 'testing' branch into an ISO with success.

**Comment by Zeioth** [comment]
I'll try it again tomorrow. @philwyettreb could you please post  the steps you followed to build the ISO please?

**Comment by Zeioth** [comment]
I think I've found an easier way

1) Copy this file
2) Run it
3) There's no step 3

[iso-builder.txt](https://github.com/Antergos/antergos-iso/files/802560/iso-builder.txt)

**Comment by philwyettreb** [comment]
@Zeioth That script just about covers it.

I am currently building iso 'testing' branch with 'colors' gfxboot branch.

![screenshot from 2017-03-03 20-27-14](https://cloud.githubusercontent.com/assets/24805379/23568546/d7df9eaa-0052-11e7-8481-a909740a797e.png)

Testing will be over this weekend. The shot above really now needs the addition of a Antergos white logo and name.

**Comment by karasu** [comment]
>The shot above really now needs the addition of a Antergos white logo and name.

I'm not much of an artist, to be honest. Feel free to change `back800x600.jpg` and `back640x480.jpg` and do a pull request in the `antergos-gfxboot` repository ;)

By the way, `nvidia`, `vesa` and `dummy` are work in progress options. They probably won't work as they are now.

**Comment by philwyettreb** [comment]
:-) Yes, I will do a little artwork and see what we can come up with. I see what I want in one of the cnchi screens.

I am building some nvidia machines to test the nvidia portions especially. I will test all options and see what explodes. :-)

**Comment by philwyettreb** [comment]
@karasu PR: https://github.com/Antergos/antergos-gfxboot/pull/3 for the background for 'colors' with nice logo added.

**Comment by karasu** [comment]
https://github.com/Antergos/antergos-gfxboot/commit/eed6e587813fb777ac65a72f76795244a04c9967

Thanks!
